OutSystems/outsystems-datagrid-reactive
# ROU-3448: Fixing export loading message

This fixes a bug where exporting messages would throw an error if Grid didn't have a name. This  PR also removes a Wijmo deprecated property and fixes a bug where the context menu was not working properly on JSONSerialized Autogen grid after #242.

### Test Steps
1. Open the context menu
2. Press export CSV/Excel

Expected: No error should be thrown.
